pairing/bn256/suite_test.go

```diff
@@ -365,3 +365,65 @@ func Test_g2_2add_oncurve_issue400(t *testing.T) {
 	err = p.UnmarshalBinary(ma)
 	require.NoError(t, err)
 }
+
+// Test doing pairings with a point produced by G2's Neg function
+// This fails with useneg == true in kyber 3.0.11.
+func testpairG2negUseneg(t *testing.T, useneg bool) {
```

Comment:
please add a third argument, kyber.Group, so you don't need a G1 and a G2 version of it, but can call it with the appropriate group.